### ISSUE TITLE
Support different decimal separators

### ISSUE THREAD

**Issue by jonmuell** [Receipt-Wrangler/receipt-wrangler-desktop#310]
English speaking countries are using a dot (.) as decimal point and a comma (,) as thousands separator. In many other countries it is the other way around, which is currently not supported by Receipt Wrangler.

As a German, I would like to be able to use the currently unsupported variant. This would allow me to enter the amount with only my numpad. The correct separator could be based on the chosen currency (easy) or browser locale (a bit more work).

Currently supported format: `1,000.99€`
Requested format: `1.000,99€`

**Comment by Noah231515** [comment]
Yeah this is a great point. I'll come up with a way for users to have full control over currency formats.

**Comment by Noah231515** [comment]
Was implemented in v5.4.0 https://github.com/Receipt-Wrangler/receipt-wrangler-api/releases/tag/v5.4.0
